Makefile

```diff
@@ -31,10 +31,15 @@ test-doc:
 ifeq ($(BITS),64)
 	$(PYTEST) $(shell find doc -name '*.rst' | sort)
 endif
+test-code-parallel: in
+	$(PYTEST) -n auto --showlocals -v sklearn --durations=20
```

Comment:
Maybe add a comment that this requires pytest-xdist. Else pytest fails with

pytest: error: unrecognized arguments: -n